packages/server/package.json

```diff
@@ -36,6 +36,8 @@
     "@osmosis-labs/types": "^1.0.0",
     "@osmosis-labs/unit": "0.10.24-ibc.go.v7.hot.fix",
     "@osmosis-labs/utils": "^1.0.0",
+    "@sentry/core": "^9.5.0",
+    "@sentry/node": "^9.5.0",
```

Comment:
What are these used for?

Reply:
Uh, I'm not sure.  My yarn dev wasn't working without them.  Warp added those to fix it